### DISCUSSION TITLE
Your browser does not support the video tag

### DISCUSSION THREAD

**Discussion by asimex** [GrapesJS/grapesjs#5867] [Q&A]
When i am importing a html template which contains a video ,I am getting the error Your browser does not support the video tag.

**Comment by mosh-tudor** [comment]
@asimex Which browser? What version? What kind of video do you have? Why don't you show us the code too, if you really need help?

**Reply by asimex** [reply]
I m importing the html code of this website
https://www.daughtercreative.com/ There is a video on this page that video is not supported.

**Reply by asimex** [reply]
I m using chrome browser you can import the html code of this website on grapes demo editor

**Reply by mosh-tudor** [reply]
Your video works fine in Chrome with the VIDEO component.

https://www.dropbox.com/s/q38jplsjbmgcbe7/Daughter_SizzleReel2021.mp4?&raw=1

<img width="871" alt="Screenshot 2024-05-09 at 13 06 59" src="https://github.com/GrapesJS/grapesjs/assets/668451/f8c66570-c0be-4cd2-a408-43090c91dce6">

---

Maybe the problem is with Dropbox, since it is not a video hosting site. Try VIMEO or YouTube.

**Comment by asimex** [comment]
Thanks for the reply.
But the real issue is happening when importing the whole website html
https://codepen.io/hereimarif/pen/bGyGvmo
when running the html code in browser getting all the contents of the page but when importing the same html in grapes js editor contents are missing video is not showing 
<img width="1440" alt="Screenshot 2024-05-10 at 10 07 23 AM" src="https://github.com/GrapesJS/grapesjs/assets/38231806/55a43f01-d2ce-41df-aabf-bb576829dd66">
<img width="1440" alt="Screenshot 2024-05-10 at 10 07 54 AM" src="https://github.com/GrapesJS/grapesjs/assets/38231806/5b22bccd-32ab-4cee-b467-d889588205f4">

**Comment by artf** [comment]
Unfortunately, the video component is really basic at the moment and only a `videos` tag with `src` is supported, not with the `source` one.
